ckan/requirements.txt

```diff
@@ -13,7 +13,7 @@ chardet==3.0.4
 -e git+https://github.com/ckan/ckan.git@eca78d5df65414249c5a76620c1acae1ddc76cc3#egg=ckan
 -e git+https://github.com/ckan/ckanext-archiver.git@4cb10ac9f96fc78bc7be4b3dee8fbc56049b2865#egg=ckanext-archiver
 -e git+https://github.com/GSA/ckanext-datagovcatalog.git@531d20c925b4b8954e71a25f992ebbf9000b4c30#egg=ckanext-datagovcatalog
--e git+https://github.com/GSA/ckanext-datagovtheme.git@a89917e07f7df106399192c79166bf92aceb0e71#egg=ckanext-datagovtheme
+-e git+https://github.com/GSA/ckanext-datagovtheme.git@32b890a7c24dfbe58f01e46f5003325d70b2fb45#egg=ckanext-datagovtheme
```

Comment:
```suggestion
-e git+https://github.com/GSA/ckanext-datagovtheme.git@master#egg=ckanext-datagovtheme
```

Reply:
@FuhuXia the requirements.txt file is self-generated. We don't suppose to change it. We just change the `toml` file